homeassistant/components/shelly/switch.py

```diff
@@ -66,6 +66,12 @@ class RpcSwitchDescription(RpcEntityDescription, SwitchEntityDescription):
     sub_key="value",
 )
 
+RPC_SCRIPT_SWITCH = RpcSwitchDescription(
+    key="script",
+    sub_key="running",
+    entity_registry_enabled_default=False,
```

Comment:
After some rethinking, I suggest these entities use the `config` category. Changing the script state is part of the device configuration.
```suggestion
    entity_registry_enabled_default=False,
    entity_category=EntityCategory.CONFIG,
```

Reply:
Back to the original code then